src/normalize.js

```diff
@@ -7,14 +7,15 @@ export const normalizeIndex = function (index) {
   const indexItems = index.map(normalizeVersion)
   const versions = getAllVersions(indexItems)
   const majors = getMajors(indexItems)
-  return { versions, majors }
+  const nodeNpmVersions = getAllNodeNpmVersions(indexItems)
```

Comment:
Instead of returning a different export `nodeNpmVersions`, what are your thoughts on adding a new field to the `versions`/`majors` exports instead? Some pros:
  - This would apply to both `versions` and `majors`
  - This would be more extensible to additional fields in the future, such as `v8` or `modules`. At the moment, I'd prefer not to add new fields unless there is a need for it to keep the interface minimal, but there might be a need.